Update and clarify Permissions-Policy usage (#42534)
mdn/content: files/en-us/web/http/reference/headers/permissions-policy/bluetooth/index.md

## Default policy

> The default allowlist for `bluetooth` is `self`. The top-level browsing context and same-origin iframes are allowed access to the `bluetooth` feature by default.

Reviewer: ```suggestion
The default allowlist for `bluetooth` is `self`. The top-level browsing context and same-origin `<iframe>` embeds are allowed access to the `bluetooth` feature by default.
```

Author: I thought about being more verbose here but ultimately decided against it, since both [existing writing on MDN](https://github.com/mdn/content/blob/26cfe9e86b714bdce65c58089f0496206b3436f3/files/en-us/web/http/reference/headers/permissions-policy/fullscreen/index.md?plain=1#L60) and [the specification itself](https://www.w3.org/TR/2025/WD-permissions-policy-1-20251006/#example-1db1f085) set a precedent for being less verbose here.

## Examples

### Basic usage

SecureCorp Inc. wants to disallow `bluetooth` within all browsing contexts except for its own origin and those whose origin is `https://example.com`. It can do so by delivering the following HTTP response header to define a Permissions Policy:

```http
Permissions-Policy: bluetooth=(self "https://example.com")
```

> Then include an {{HTMLElement('iframe','allow','#Attributes')}} attribute on the `<iframe>` element:

Author: Both the HTTP header and iframe `allow` attribute must be used simultaneously for this example (see prior discussion).

> Then include an {{HTMLElement('iframe','allow','#Attributes')}} attribute on the `<iframe>` element:

Reviewer: Having looked at the example again, the description still seems a bit terse and brief to me. I think it would also help if you updated the first paragraph to specify that the usage of the `https://example.com` origin is going to be limited to embeds (I know this is implied, but it still came as a surprise later on).

Maybe update "...within all browsing contexts except for its own origin and those whose origin is `https://example.com`." to something like "...within all browsing contexts except for its own origin and embedded pages whose origin is `https://example.com`."?

```html
<iframe src="https://example.com/blue" allow="bluetooth"></iframe>
```

> [!NOTE]

> > Specifying the `Permissions-Policy` header in this manner disallows `bluetooth` for other origins, even if they would be allowed by the `<iframe>` `allow` attribute.

Reviewer: ```suggestion
> Specifying the `Permissions-Policy` header in this manner disallows `bluetooth` for other origins, even if they were allowed by the `<iframe>` `allow` attribute.
```
